src/hotspot/share/gc/shared/objectCountClosure.hpp

```diff
@@ -0,0 +1,29 @@
+#ifndef SHARE_GC_SHARED_OBJECTCOUNTCLOSURE_HPP
+#define SHARE_GC_SHARED_OBJECTCOUNTCLOSURE_HPP
+
+#include "gc/shared/gcId.hpp"
+#include "gc/shared/objectCountEventSender.hpp"
+#include "jfr/jfrEvents.hpp"
+#include "memory/allStatic.hpp"
+#include "runtime/mutex.hpp"
+#include "memory/heapInspection.hpp"
+#include "utilities/macros.hpp"
+#include "utilities/ticks.hpp"
+
+class KlassInfoEntry;
+class Klass;
+
+class ObjectCountClosure : public AllStatic {
+    static KlassInfoTable* cit;
+
+public:
```

Comment:
Please provide a documentation comment for all the public APIs in every class you write.